### PR TITLE
feat(Confirm): add more props, button-ok and button-cancel slots

### PR DESCRIPTION
**Pull request by albertpratomo**

### CODE REVIEW

File: src/components/Confirm/Confirm.vue [hunk 1]
```diff
@@ -1,24 +1,81 @@
...
+    },
+});
+
+const emit = defineEmits<{(e: 'confirmed'): void}>();
+
+const ok = () => {
+    emit('confirmed');
+
+    hide();
+};
```
Comment (TimDirks) [inline src/components/Confirm/Confirm.vue:72]: Might be useful to have a cancelled emit as well

```suggestion
const emit = defineEmits<{
    (e: 'cancelled'): void,
    (e: 'confirmed'): void,
}>();

const cancel = () => {
    emit('cancelled');

    hide();
};

const ok = () => {
    emit('confirmed');

    hide();
};
```

File: src/components/Confirm/Confirm.vue [hunk 2]
```diff
@@ -1,24 +1,81 @@
...
+    title: {
+        type: String,
+        default: 'Confirmation',
+    },
+});
+
+const emit = defineEmits<{(e: 'confirmed'): void}>();
+
+const ok = () => {
+    emit('confirmed');
```
Comment (TimDirks) [inline src/components/Confirm/Confirm.vue:69]: Missing tests for the `confirmed` emit

**Review by TimDirks — APPROVED** [review]

File: src/components/Confirm/Confirm.vue (hunk 1 above)
Comment (albertpratomo) [reply src/components/Confirm/Confirm.vue:72]: goed idee

File: src/components/Confirm/Confirm.vue [hunk 3]
```diff
@@ -1,24 +1,92 @@
...
+    hide();
+};
+
+const modal = ref<ComponentPublicInstance<typeof Modal>>();
+
+const hide = () => modal.value.bsModal?.hide();
+
+const show = () => modal.value.bsModal?.show();
+
+defineExpose({hide, show});
```
Comment (thomasowow) [inline src/components/Confirm/Confirm.vue:91]: Is is also exposing `bsModal` automatically?

**Review by thomasowow — APPROVED** [review]

File: src/components/Confirm/Confirm.vue (hunk 3 above)
Comment (albertpratomo) [reply src/components/Confirm/Confirm.vue]: No, but we exposes `hide, show, toggle` which are all you need in a modal

**Pull request merged**